docs/_layouts/default.html

```diff
@@ -80,7 +80,7 @@
 
           <li>
             <span class="nav-header">
-              <a href="{{ '/tutorials' | relative_url }}" class="center-vertical">Tutorials</a>
+              <a href="{{ '/tutorials' | relative_url }}" class="center-vertical">Tutorials (optional)</a>
```

Comment:
Am also considering changing "Tutorials" to "Hands-on Wash" so the title is less intimidating. Latter's a bit more involved though.

Reply:
@MikaelSmith What are your thoughts on this?

Reply:
I think I'd stick with Tutorials, not clear to me that Hands-on Wash is really better.

Reply:
Tutorials it is.